Readme.md

```diff
@@ -12,22 +12,35 @@ This is a GCC codegen for rustc, which means it can be loaded by the existing ru
 The primary goal of this project is to be able to compile Rust code on platforms unsupported by LLVM.
 A secondary goal is to check if using the gcc backend will provide any run-time speed improvement for the programs compiled using rustc.
 
-### Dependencies
-
-**rustup:** Follow the instructions on the official [website](https://www.rust-lang.org/tools/install)
-
-**DejaGnu:** Consider to install DejaGnu which is necessary for running the libgccjit test suite. [website](https://www.gnu.org/software/dejagnu/#downloading)
-
-
+## Getting Started
 
-## Building
+Note: **This requires a patched libgccjit in order to work.
+You need to use my [fork of gcc](https://github.com/antoyo/gcc) which already includes these patches.**
```

Comment:
To make this clearer, perhaps mention that the default example config download libgccjit from the CI so this will use the fork automatically.